docs/api/structures/offscreen-shared-texture.md

```diff
@@ -0,0 +1,16 @@
+# OffscreenSharedTexture Object
+
+* `textureInfo` Object - The shared texture info. Normally will be released automatically when the callback ends.
+To use in async or share this texture with another process, you need to call `takeReleaser` to take the ownership of the texture.
+  * `widgetType` string - The widget type of the texture, could be `popup` or `frame`.
+  * `pixelFormat` string - The pixel format of the texture, could be `rgba` or `bgra`.
+  * `sharedTextureHandle` string - _Windows_ _macOS_ The handle to the shared texture.
+  * `planes` Object[] - _Linux_ Each plane's info of the shared texture.
+    * `stride` number - The strides and offsets in bytes to be used when accessing the buffers via a memory mapping. One per plane per entry.
+    * `offset` number - The strides and offsets in bytes to be used when accessing the buffers via a memory mapping. One per plane per entry.
+    * `size` number - Size in bytes of the plane. This is necessary to map the buffers.
+    * `fd` number - File descriptor for the underlying memory object (usually dmabuf).
+  * `modifier` string - _Linux_ The modifier is retrieved from GBM library and passed to EGL driver.
```

Comment:
According to chromium it will work on X11 and Wayland. However I only tested on Wayland but I think it is possible to work on any display server that supports gbm? These params are general on Linux.